### PR TITLE
Merge CI branch into master

### PR DESCRIPTION
**Pull request by s0undt3ch**

### CODE REVIEW

File: .drone.jsonnet [hunk 1]
```diff
@@ -1,6 +1,6 @@
 local distros = [
   // Multiprier is way to throttle API requests in order not to hit the limits
```
Comment (bryceml) [inline .drone.jsonnet]: can we fix the misspelling of Multiplier?

File: Makefile [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 OS  =
 OS_REV  =
+SALT_BRANCH = develop
```
Comment (bryceml) [inline Makefile]: should this be master?

File: tasks/build.py [hunk 3]
```diff
@@ -38,9 +38,11 @@ def build_aws(ctx,
               distro,
               distro_version=None,
               region='us-west-2',
+              salt_branch='develop',
```
Comment (bryceml) [inline tasks/build.py]: should this be master?

File: tasks/build.py [hunk 4]
```diff
@@ -92,15 +106,20 @@ def build_aws(ctx,
...
+                                                                salt_branch,
+                                                                build_template)
     ctx.run(cmd, echo=True, env={'PACKER_TMP_DIR': packer_tmp_dir})
 
 
 @task
 def build_docker(ctx,
                  distro,
                  distro_version=None,
+                 salt_branch='develop',
```
Comment (bryceml) [inline tasks/build.py]: should this be master?

File: Makefile [hunk 5]
```diff
@@ -26,10 +27,10 @@ validate: check-paths
...
-	@exit 1
+	$(error Please run 'inv build-aws --distro $(OS) --distro-version $(OS_REV) --salt-branch=$(SALT_BRANCH)' instead.)
++	@exit 1
 
 .PHONY: build-staging
 build-staging: check-paths
-	$(error Please run 'inv build-aws --distro $(OS) --distro-version $(OS_REV) --staging' instead.)
-	@exit 1
+	$(error Please run 'inv build-aws --distro $(OS) --distro-version $(OS_REV) --salt-branch=$(SALT_BRANCH) --staging' instead.)
++	@exit 1
```
Comment (bryceml) [inline Makefile]: I thought another recent pr got rid of the `+`

File: Makefile [hunk 6]
```diff
@@ -26,10 +27,10 @@ validate: check-paths
 
 .PHONY: build
 build: check-paths
-	$(error Please run 'inv build-aws --distro $(OS) --distro-version $(OS_REV)' instead.)
-	@exit 1
+	$(error Please run 'inv build-aws --distro $(OS) --distro-version $(OS_REV) --salt-branch=$(SALT_BRANCH)' instead.)
++	@exit 1
```
Comment (bryceml) [inline Makefile]: I thought another recent pr got rid of the `+`

File: Docker/debian/debian.json [hunk 7]
```diff
@@ -27,9 +27,87 @@
       "inline": [
         "rm -rf /etc/apt/apt.conf.d/20auto-upgrades",
         "apt-get update -y && apt-get upgrade -y",
-        "apt-get install -y systemd git vim ",
+        "apt-get install -y systemd git vim sudo openssh-server curl",
         "rm -rf /tmp/* /var/lib/apt/lists/* && apt-get clean"
```
Comment (bryceml) [inline Docker/debian/debian.json:31]: Long term we should put
```bash 
rm -rf /tmp/* /var/lib/apt/lists/* && apt-get clean
```
at the end, same for other distros.  We should also probably delete the saltstack repo files at the end in the same place as this.  I don't know if you want to do that in this pr or another one later.

**Comment by bryceml** [comment]
Long term we want to just have these be files in salt-jenkins.  Is this going to just be a step in getting there?

File: Docker/debian/debian.json (hunk 7 above)
Comment (s0undt3ch) [reply Docker/debian/debian.json:31]: On another one later, or in salt-jenkins

**Comment by s0undt3ch** [comment]
This is a step to get us there

**Comment by s0undt3ch** [comment]
Let's wait on merging this in though.

**Pull request closed without merging**